src/App.js

```diff
@@ -83,7 +85,8 @@ const App = ({ stateCode, data }) => {
 
   const { flowData, lastDate, comparedToDate } = generateFlowDiagramData(stateMetricData);
 
-  const keyInsightsData = generateKeyInsightsData(flowData);
+  const correctionsKeyInsightsData = generateKeyInsightsData(flowData);
+  const jailsKeyInsightsData = generateJailsKeyInsightsData(jailsMetricData);
```

Comment:
In cases where we are creating a new parallel function for _jails_ data, we should rename the existing corrections-focused function. So we should rename `generateKeyInsightsData` to `generateCorrectionsKeyInsightsData`.

Reply:
Fixed